package-lock.json

```diff
@@ -0,0 +1,7998 @@
+{
+  "name": "cloudinary",
```

Comment:
@rebeccapeltz , please don't commit package-lock files

Reply:
If not committing package-lock.json, should you add to .gitignore?   
If interested in letting github automate security updates here's an article from them https://help.github.com/en/github/managing-security-vulnerabilities/configuring-automated-security-updates